### DISCUSSION TITLE
Delete row from 'Display column'

### DISCUSSION THREAD

**Discussion by gmwill934** [TanStack/table#4257] [Q&A]
Hello

Is it possible to delete a row from a `Display Column`? 

I have declared my `column defs`  with a [display column](https://tanstack.com/table/v8/docs/guide/column-defs#column-def-types)

```ts
const columnHelper = createColumnHelper<SomeType>();

const columnDefinitions = [
  // Accessor Columns
    columnHelper.accessor...
  // Display Column
  columnHelper.display({
    id: 'delete',
    header: 'Actions',
    cell: (info) => (
      <button className='p-1 bg-red-500 hover:bg-red-700 active:bg-red-800 cursor-pointer transition-all duration-300'>
        <TrashIcon
          className='h-5 w-5 text-white'
          onClick={() => console.log(`Deleting row with id of ${info.row.id}`)}
        />
      </button>
    ),
  }),
]
```

What I want to achieve is fully delete a row from my table when the user presses the button. I have not found a quick and efficient way on doing this. Does `react-table` exposes something I can use?

Is this possible?

**Comment by mmhuntsberry** [comment]
@gmwill934 

I've been looking for a solution to this as well, any progress or updates you've found?
